.github/workflows/deploy-site-reusable.yaml

```diff
@@ -115,6 +105,12 @@ jobs:
           git config user.name "ASF Logging Services RM"
           git config user.email private@logging.apache.org
 
+      # Checking out a new branch will delete the `node_modules` folder
+      - name: Save Node.js modules
+        shell: bash
+        run: |
+          zip -qX "$RUNNER_TEMP"/node_modules.zip node_modules
```

Comment:
Shall we add `-0` to speed things up a bit?

Reply:
Fixed in https://github.com/apache/logging-parent/pull/367/commits/c5229479cbe1b1332f088d0eff4c481582103723